app/models/template.rb

```diff
@@ -31,10 +31,6 @@ def self.template_includes
     []
   end
 
-  def preview_host_collection
```

Comment:
I'm afraid this change will break template previewing for Job templates used in remote execution plugin. Note that there we also want to be able to preview templates for unmanaged hosts.